connectivity/drivers/ble/FEATURE_BLE/COMPONENT_BlueNRG_2/mbed_lib.json

```diff
@@ -10,6 +10,10 @@
         "valid-public-bd-address": {
             "help": "Read the BD public address at startup",
             "value": false
+        },
+        "rpa_swap_workaround": {
```

Comment:
In my opinion, the flag belongs to the cordio configuration then it can be enabled for BlueNRG2, WB55, ...

Reply:
OK but how do you detect the presence of BLueNRG2?